model/clusters_mgmt/v1/azure_cluster_capabilities.model

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct AzureClusterCapabilities {
+    // Disabled capabilities
+    Disabled []AzureOptionalClusterCapability
+}
```

Comment:
This will need to change to remove the `Azure` prefix.

Reply:
Done.